### PR TITLE
added style and layout for the entire signin view:rocket:

### PR DESCRIPTION
**Pull request by Segun-Ade**

### CODE REVIEW

File: src/components/vendorSignin/components/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+ * @param {boolean} isSignedIn is set to true
+ * when a user is successfully signed in and
+ * false if otherwise.
+ */
+function FormCard(info, isSignedIn) {
+    if (isSignedIn === true) {
+        return <Redirect to="/dashboard" />;
+    }
+    return FormComponent(info);
+}
```
Comment (enyeInc) [inline src/components/vendorSignin/components/Form.jsx]: You do not need this component

File: src/components/vendorSignin/components/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+function SigninForm() {
+    const data = useSelector(state => state);
+    const {
+        vendorSignin: { email, password, isSignedIn },
+    } = data;
+
+    const info = { email, password };
+    return FormCard(info, isSignedIn);
```
Comment (enyeInc) [inline src/components/vendorSignin/components/Form.jsx]:
```
return isSignedIn === true ? <Redirect to="/dashboard" /> : FormComponent(info);
```

File: src/components/vendorSignin/components/FormCard.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+ *
+ * @constant
+ */
+const renderSignUpView = (
+    <div className="sign-up-feature">
+        <p>
+            {SIGN_UP_TEXT}
+            <Link to={SIGN_UP_URL}>{SIGN_UP_LINK_TEXT}</Link>
+        </p>
+    </div>
```
Comment (enyeInc) [inline src/components/vendorSignin/components/FormCard.jsx]: add the class to the `p` tag and remove the `div`

File: src/components/vendorSignin/components/FormCard.jsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+ * @constant
+ */
+const renderVendorSignInView = (
+    <div className="sign-up-feature">
+        <p>
+            {VENDOR_SIGN_IN_TEXT}
+            <Link to={VENDOR_SIGN_IN_URL}>{VENDOR_SIGN_IN_LINK_TEXT}</Link>
+        </p>
+    </div>
+);
```
Comment (enyeInc) [inline src/components/vendorSignin/components/FormCard.jsx]: same here

File: src/components/vendorSignin/components/Signin.jsx [hunk 5]
```diff
@@ -1,7 +1,10 @@
 import React from 'react';
+import FormView from './FormCard';
 
 const VendorSignin = () => (
-    <div />
+    <div>
+        <FormView />
+    </div>
 );
```
Comment (enyeInc) [inline src/components/vendorSignin/components/Signin.jsx]: this component is not doing anything other than return `FormView`. It is not needed

File: src/components/vendorSignin/components/Signi.jsx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+// import React from 'react';
+// import FormView from './Signin';
```
Comment (edbella) [inline src/components/vendorSignin/components/Signi.jsx]: Why not delete this file if its not needed?

Comment (Segun-Ade) [reply src/components/vendorSignin/components/Signi.jsx]: That was an oversight, I intended to

Comment (Segun-Ade) [reply src/components/vendorSignin/components/Signi.jsx]: I'll do that right away

File: src/components/vendorSignin/components/Form.jsx [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+/**
+ *
+ * @function {fuction} dispatch trigers the 'rememberMe'
+ * action when the checkbox is checked or unchecked.
+ */
+function checkBox(dispatch) {
+    return (
+        <Checkbox onChange={e => dispatch(rememberMe(e.target.value))}>
+            {REMEMBER_ME}
```
Comment (olajohn-ajiboye) [inline src/components/vendorSignin/components/Form.jsx]: Add the params for this function

**Pull request merged**